Merchant/App/PaymentInput/PaymentInputViewController.swift

```diff
@@ -11,26 +11,26 @@ import BDCKit
 
 class PaymentInputViewController: PinViewController {
     
-    var amountLabel: BDCLabel = {
+    let amountLabel: BDCLabel = {
         let label = BDCLabel.build(.header)
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
-    var amountView: UIView = {
+    let amountView: UIView = {
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possible many more places than just these...